src/main/java/org/snomed/snowstorm/core/data/services/QueryService.java

```diff
@@ -7,6 +7,7 @@
 import it.unimi.dsi.fastutil.longs.LongArraySet;
 import it.unimi.dsi.fastutil.longs.LongComparators;
 import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.search.sort.FieldSortBuilder;
```

Comment:
Please remove all the changes in this class, they are incorrect and not needed.

Reply:
This was left over from a previous change attempt - removed.